aggregating-profiler-prometheus/build.gradle.kts

```diff
@@ -10,15 +7,14 @@ dependencies {
     /**
      * Runtime
      */
-    compile(Libs.slf4j_api)
-
-    compile(project(":aggregating-profiler"))
+    api(Libs.slf4j_api)
+    api(project(":aggregating-profiler"))
+    api(Libs.kotlin_logging)
```

Comment:
same question about logging library necessity

Reply:
removed dependency from api configuration (but not from testImplementation)